<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1438</width>
    <height>806</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>TYDEA Pro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../TYDEA_logo.png</normaloff>../TYDEA_logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="LeftVerticalLayout">
        <item>
         <widget class="QGroupBox" name="reverseWhoisGenerationGroupBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Reverse Whois Generation from  Keywords</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="1">
            <widget class="QCheckBox" name="useVirustotalCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="useVirustotalLabel">
             <property name="text">
              <string>Use Virustotal resolver</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="keywordsListLabel">
             <property name="text">
              <string>Keywords list</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="FilePathSelector" name="keywordsListSelectorWidget" native="true"/>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="reverseWhoisGenerateButton">
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>From :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCalendarWidget" name="reverseWhoisCalendar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="firstDayOfWeek">
              <enum>Qt::Monday</enum>
             </property>
             <property name="gridVisible">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QCalendarWidget::SingleSelection</enum>
             </property>
             <property name="horizontalHeaderFormat">
              <enum>QCalendarWidget::ShortDayNames</enum>
             </property>
             <property name="verticalHeaderFormat">
              <enum>QCalendarWidget::NoVerticalHeader</enum>
             </property>
             <property name="navigationBarVisible">
              <bool>true</bool>
             </property>
             <property name="dateEditEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>To :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCalendarWidget" name="endCalendarWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="firstDayOfWeek">
              <enum>Qt::Monday</enum>
             </property>
             <property name="verticalHeaderFormat">
              <enum>QCalendarWidget::NoVerticalHeader</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="typosquattingGroupBox">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Typosquatting Generation from Domain</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="1">
            <widget class="QCheckBox" name="TLDCheckBox">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="dnstwitstCheckBox">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="domains"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="urlcrazyLabel">
             <property name="text">
              <string>Urlcrazy</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="urlcrazyCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="userFunctionsLabel">
             <property name="text">
              <string>User function</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="dnstwistLabel">
             <property name="text">
              <string>Dnstwist</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="enterDomainsLabel">
             <property name="text">
              <string>Enter domains</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="FilePathSelector" name="userFunctionsSelectorWidget" native="true"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="useAllTLDsLabel">
             <property name="text">
              <string>Use all TLDs</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="QPushButton" name="typosquattingGenerateButton">
             <property name="text">
              <string>Generate</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="commandGroupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QPushButton" name="New">
             <property name="text">
              <string>New Database</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="Load">
             <property name="text">
              <string>Load Database</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="adduserlistButton">
             <property name="text">
              <string>Add Userlist</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="checkDNSButton">
             <property name="text">
              <string>Check DNS</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="analyseButton">
             <property name="text">
              <string>Analyse</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="loadedDatabaseLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DatabaseManagmentLayout"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableView" name="keywordsTableView"/>
          </item>
          <item>
           <widget class="QTableView" name="domainsTableView"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="RightVerticalLayout">
        <item>
         <widget class="QTableView" name="typosquatDomainsTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addToWhitelistButton">
            <property name="text">
             <string>Toggle whitelist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addToBlacklistButton">
            <property name="text">
             <string>Toggle blacklist</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1438</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilePathSelector</class>
   <extends>QWidget</extends>
   <header location="global">filePathSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
